packages/nodes-base/nodes/DebugHelper/DebugHelper.node.json

```diff
@@ -9,5 +9,16 @@
 				"url": "https://docs.n8n.io/integrations/builtin/core-nodes/n8n-nodes-base.debughelper/"
 			}
 		]
-	}
+	},
+	"alias": [
+		"Mock",
+		"Sample",
+		"Demo",
+		"Test",
+		"Throw error",
+		"OOM",
+		"Out of Memory",
+		"placeholder",
+		"do nothing"
```

Comment:
Don't we want NoOp node to show up first for these last 2?

Reply:
NoOp doesn't have placeholder as an option or "do nothing" but I have removed "do nothing" from this node as chances are people wouldn't search for that and we can always add it later if we need it.